### PR TITLE
Automated Test - COMMERCE-5869 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
https://issues.liferay.com/browse/COMMERCE-5869, CC:@lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
Hi @PauloVAS, please uncomment your teardown and remove the **@ignore = true**. Thank you!

**Comment by PauloVAS** [comment]
@brittneyq fixed.

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -190,12 +190,45 @@ definition {
...
+
+		Check.checkToggleSwitch(
+			key_productName = "U-Joint",
+			locator1 = "CommerceEntry#SELECT_CHECKBOX");
+
+		Refresh();
+
+		AssertElementPresent(
+			key_productName = "U-Joint",
+			locator1 = "CommerceEntry#SELECT_CHECKBOX");
```
[the remark this answers is not in the document]
Comment (PauloVAS) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: @lorenzolapescara I tried to use only the "AssertChecked" function, but i was always getting the same error, so i decided to use "AssertChecked.assertCheckedNotVisible" and it worked 100%.

**Comment by PauloVAS** [comment]
@lorenzolapescara fixed

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**